### PR TITLE
[fme-detect] Introduce fme-detect

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit introduces fme-detect module.

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/fme-detect/src/EqualizePatternFinder.cpp [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    {
+      f.pre_scale = true;
+      f.post_scale = true;
+      f.post_shift = true;
+      // NOTE CircleTransposeConv does not support fused activation function
+    }
+    return f;
+  }
+
+  Fusability visit(luci::CircleDepthwiseConv2D *node)
```
Comment (dayo09) [inline compiler/fme-detect/src/EqualizePatternFinder.cpp:74]:
```suggestion
  Fusability visit(const luci::CircleDepthwiseConv2D *node)
```

How about using const parameter?

Comment (mhs4670go) [reply compiler/fme-detect/src/EqualizePatternFinder.cpp:74]: Hmm.. it's been inherited from `CircleNodeMutableVisitorBase`. 

@jinevening Is there any reason?

File: compiler/fme-detect/src/EqualizePatternFinder.cpp [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+struct Fusability
+{
+  bool pre_scale = false;
+  bool post_scale = false;
+  bool pre_shift = false;
+  bool post_shift = false;
+};
+
+struct GetFusability final : public luci::CircleNodeMutableVisitor<Fusability>
```
Comment (dayo09) [inline compiler/fme-detect/src/EqualizePatternFinder.cpp:37]: How about spliting this part..? To avoid frequent merge conflict in near future?

**Review by jinevening — APPROVED** [review]
LGTM

One concern is that I'm not sure it is better to include `shift`

**Pull request merged**